ipatests/test_webui/ui_driver.py

```diff
@@ -1980,3 +2006,27 @@ def assert_notification(self, type='success', assert_text=None):
         assert is_present, "Notification not present"
         if assert_text:
             assert assert_text in is_present.text
+
+    def assert_last_error_dialog(self, text_tba, details=False,
```

Comment:
`text_tba` takes some time to decipher. Wouldn't e.g. `expected_err` or `assert_text` as used elsewhere be better?

Reply:
fixed